RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerViewModel.cs

```diff
@@ -577,6 +577,23 @@ public Visibility EmptyUIRefreshMessageVisibility
             }
         }
 
+        public void Search(IEnumerable<CodeExplorerItemViewModel> nodes, string searchString)
```

Comment:
This is more than a `Search` though, the name doesn't convey the side-effects the method has - e.g. I'd expect a `Search` method to return the search results or something. Seems `FilterByName` would be more descriptive. From the user's point of view that can remain a "search box" though.

Reply:
You're right.